### DISCUSSION TITLE
Donations

### DISCUSSION THREAD

**Discussion by dresslerc** [PeteRager/lennoxs30#80] [General]
Thank you for your efforts with this project.  My Lennox system with its variable heat pump has been amazing but their software has been trash. 

You solution let's me bypass the Lennox app all together which is amazing!

Please setup a place where we can donate!

**Comment by PeteRager** [comment]
Thanks for the kind words.  I’ll get something setup.

**Comment by PeteRager** [comment]
[!["Buy Me A Coffee"](https://www.buymeacoffee.com/assets/img/custom_images/orange_img.png)](https://www.buymeacoffee.com/PeteRage)
